module1/homework/grayscaleImages/test.cpp

```diff
@@ -1,9 +1,9 @@
 #include <algorithm>
 #include <array>
-#include <utility>  // for std::pair<>
+#include <utility>
```

Comment:
You can remove array and vector include since you have included them in the header.